README.md

```diff
@@ -19,6 +19,52 @@ The `.cursorrules` file is our attempt at fixing the common mistakes the AI assi
 
 Also, make sure you have the [Wasp docs](https://wasp-lang.dev/docs) indexed in Cursor. You can do this by going to `preferences > cursor settings > features > add new doc `. Then, you can include them in Cursor chat by using the `@docs` keyword. This often improves the AI's ability to help you with Wasp-specific code.
 
+## Project Structure
+
+The project is structured to offer just enough context for the AI to help you with Wasp-specific code, and to get you started with building your app quickly, but without overwhelming you with too much code.
+
+In the sections below, we'll go through each directory and explain why we've included them and what they do.
+
+```
+src/
+├── auth/                          // Auth-related files
+├── client/                        // Client-only components and the Main.tsx app wrapper
+├── exampleNotesFeature/           // Example feature code
+│   ├── ExampleNotePage.tsx           
+│   ├── ExampleNoteDashboard.tsx      
+│   └── operations.ts              // Example server-side functions 
+├── main.wasp                      // App configuration file where you can define routes, auth, operations, etc.
+├── .cursorrules                   // Cursor rules file / where the magic happens
+```
+
+### `main.wasp`
+
+The `main.wasp` file is the main configuration file for your app. It's where you define your routes, auth, operations, etc.
+
+This is Wasp's secret sauce and allows Wasp to generate the full-stack code for your app, so you can just focus on writing the business logic.
+
+For more info on Wasp's config file and how Wasp works, check out the [Wasp Introduction](https://wasp-lang.dev/docs#so-what-does-the-code-look-like).
+
+### Auth
+
+The `auth` directory contains the login and signup pages. They import and take advantage of Wasp's Auth API to handle the full-stack auth logic for you. E.g., after defining the Auth methods you'd like to use in `main.wasp`, Wasp will generate the login and signup forms for the defined methods, and handle the full-stack auth logic for you, including validation, error handling, etc.
+
+For more info on Wasp Auth, check out the [Wasp Auth docs](https://wasp-lang.dev/docs/auth/overview).
+
+### Client
+
+The `client` directory contains the client-only components and the `Main.tsx` app wrapper. It also contains the `Main.css` file, which imports Tailwind CSS so you can can get started styling your app with Tailwind without having to configure it yourself.
+
```

Comment:
You went quite detailed here! I would probably just give them a single link to Wasp docs and tell them to learn more about stuff like auth, main.wasp and similar there. But if you think this is the right amount of info, why not, that is also ok.

Reply:
I assumed it would help people get started faster...